bin/templatemap_generator.php

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
```

Comment:
But where should the template map generator be placed then? It doesn't make sense to add the zendframework/zendframework package just to get the template map generator. Create a new package just for the generator? Seems excessive to me... ;-)